src/nni_manager/training_service/local/localTrainingService.ts

```diff
@@ -355,7 +355,8 @@ class LocalTrainingService implements TrainingService {
         this.log.info('Stopping local machine training service...');
         this.stopping = true;
         for (const stream of this.jobStreamMap.values()) {
-            stream.destroy();
+            stream.end(0)
```

Comment:
could be better to add the github issue. what do you think @chicm-ms @SparkSnail ?

Reply:
As discussed with @chicm-ms , I've add related issue in code.